sdk/elixir/lib/dagger/core/query_builder.ex

```diff
@@ -18,12 +18,23 @@ defmodule Dagger.Core.QueryBuilder.Selection do
     }
   end
 
-  def arg(%__MODULE__{args: args} = selection, name, value) when is_binary(name) do
+  # TODO: Remove me.
```

Comment:
Did you mean to remove this now, or in a follow-up PR?

Reply:
I need to tackle it on the PR while I rework on the query builder.